app/services/bgs/dependent_service.rb

```diff
@@ -70,45 +50,95 @@ def submit_686c_form(claim)
         # detailing this specific file number / SSN error, and is therefore unable
         # to distinguish this error from others in our Sentry dashboards. That is
         # why I am deliberately raising these errors here.
-        validate_file_number_format!(file_number:)
-        validate_file_number_matches_ssn!(file_number:)
-        submit_form_job_id = if claim.submittable_686?
-                               BGS::SubmitForm686cJob.perform_async(
-                                 uuid, @icn, claim.id, encrypted_vet_info
-                               )
-                             else
-                               BGS::SubmitForm674Job.perform_async(
-                                 uuid, @icn, claim.id, encrypted_vet_info
-                               )
-                             end
+
+        perform_file_number_validations!(file_number:)
+        submit_form_job_id = submit_to_standard_service(claim:, encrypted_vet_info:)
         Rails.logger.info('BGS::DependentService succeeded!', { user_uuid: uuid, saved_claim_id: claim.id, icn: })
       end
 
       {
-        submit_pdf_job_id:,
         submit_form_job_id:
       }
     rescue => e
       Rails.logger.error('BGS::DependentService failed!', { user_uuid: uuid, saved_claim_id: claim.id, icn:, error: e.message }) # rubocop:disable Layout/LineLength
       log_exception_to_sentry(e, { icn:, uuid: }, { team: Constants::SENTRY_REPORTING_TEAM })
+
       raise e
     end
-    # rubocop:enable Metrics/MethodLength
 
     private
 
     def service
       @service ||= BGS::Services.new(external_uid: icn, external_key:)
     end
 
+    def submit_pdf_job(claim:, encrypted_vet_info:)
+      VBMS::SubmitDependentsPdfJob.perform_sync(
+        claim.id,
+        encrypted_vet_info,
+        claim.submittable_686?,
+        claim.submittable_674?
+      )
+      # This is now set to perform sync to catch errors and proceed to CentralForm submission in case of failure
+    rescue => e
+      submit_to_central_service(claim:)
+
+      raise e
+    end
+
+    def submit_to_standard_service(claim:, encrypted_vet_info:)
+      if claim.submittable_686?
+        BGS::SubmitForm686cJob.perform_async(
+          uuid, icn, claim.id, encrypted_vet_info
+        )
+      else
+        BGS::SubmitForm674Job.perform_async(
+          uuid, icn, claim.id, encrypted_vet_info
+        )
+      end
+    end
+
+    def submit_to_central_service(claim:)
+      vet_info = JSON.parse(claim.form)['dependents_application']
+
+      if Flipper.enabled?(:dependents_central_submission)
```

Comment:
I thought this flipper got removed. 

we can either remove it now, or I can create a separate ticket.